app/controllers/api/base_controller.rb

```diff
@@ -35,7 +35,7 @@ class BaseController < ActionController::Base
     rescue_from Foreman::MaintenanceException, :with => :service_unavailable
 
     def get_resource
-      instance_variable_get :"@#{resource_name}" or raise 'no resource loaded'
+      instance_variable_get(:"@#{resource_name}") || (raise 'no resource loaded')
```

Comment:
adding the brackets to the method call, i.e. `|| raise('no resource loaded')` is probably more typical